students/srmorehouse/Lesson02/series.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+
+"""
+fibonacci
+
+input: n : series in the fibonacci series
+
+return: if 0, 0
+        if 1, 1
+        else (n-2)+(n-1)
+
+No negative number error checking
+"""
+def fibonacci (n):
+  if n==0:
+    retVal=0
+  elif n==1:
+    retVal=1
+  else:
+    retVal=fibonacci(n-2)+fibonacci(n-1)
+  return retVal
```

Comment:
Styling issues:
* indentation issue: should be 4 spaces
* variable names should be snake_case
* spacing issue -> `if n == 0` or `ret_val = fibonacci(n - 2) + fibonacci(n - 1)`